index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="wrapper">
+        <div class="nav-bar">
+            <div class="logo">Charles</div>
```

Comment:
- [ ] The logo should be clickable so it would be better if you use the `<a>` tag instead of `<div>` that will  make your logo clickable and also reachable by keyboard for people that can't use the mouse

Reply:
- [ ] please try to make your portfolio responsive in mobile screens right it has some white spaces I encourage you to fix that 
![image](https://user-images.githubusercontent.com/61790587/170371370-7ca33b30-e4d7-432c-af65-91ba507d9b01.png)

Reply:
done